pkg/apis/garden/v1beta1/types.go

```diff
@@ -178,10 +178,6 @@ type OpenStackProfile struct {
 	// DNSServers is a list of IPs of DNS servers used while creating subnets.
 	// +optional
 	DNSServers []string `json:"dnsServers,omitempty"`
-	// DHCPDomain is the dhcp domain of the OpenStack system configured in nova.conf. Only meaningful for
-	// Kubernetes 1.10.1+. See https://github.com/kubernetes/kubernetes/pull/61890 for details.
-	// +optional
-	DHCPDomain *string `json:"dhcpDomain,omitempty"`
```

Comment:
Don't remove the field entirely without a proper deprecation period. Mark it rather as "deprecated" and don't use it anymore anywhere.